TIP-0003/README.md

```diff
@@ -0,0 +1,273 @@
+
+    $$sin - burned = sout$$
+
+### Transacting with Assets
+
+Transaction with assets are performed in the same way as transactions with LVLs. The only difference is that assets allow for different degrees of fungibility. At a chain level two assets are fungible if they can both be stored in the same UTXO. This poses a problem when we deal with two assets that are fungible at one level but not at the other. 
+
+#### Moving Assets
+
+Moving assets does not require to attach any extra metadata to the transaction. Each time an asset is moved, it is the user's responsibility to populate both the metadata and commitment field. It is also responsibility of the user to ensure that the metadata conforms to the data specification in the policy. The node does not enforce this. The following validations are performed by the node:
+
+- *Check Assets Fungibility:* Let $g$ be a group identifier and $s$, $s_0$, ..., $s_{n - 1}$  be series identifiers,  all of the following statements must be true:
+  - if $s$ is group and series fungible, then the number of tokens with identifier ($g$ ,$s$) in the input is equal to the number of tokens with identifier ($g$ ,$s$) in the output.
+  
+  - if $s$ is a series fungible series, and $G$ the set of group identifiers, then the sum of all tokens with identifiers ($g$, $s$), where $g \in G$, in the input is equal to the number of tokens with identifiers ($g$, $s$), where $g \in G$.
+  
+  - if $g$ is a group identifier and $S$ is the set of group fungible series, then the sum of all tokens with identifiers ($g$, $s$), where $s\in S$, in the input is equal to the number of tokens with identifier ($g$, $s$), where $s\in S$.
+  
+- *Check Assets Quantity Descriptors*: Quantity descriptors regulate how UTXOs of an asset are split or merged. This is different from alloys, as the assets being merged or split always share the same group and series identifier. Let ($g$, $s$) be an asset identifier, all the following statements must be true:
+  - If  ($g$, $s$)'s quantity descriptor is `immutable`, then, for each input UTXO containing  ($g$, $s$) with quantity $n$ , there must be exactly one output containing  ($g$, $s$) with quantity $n$.
+  - If  ($g$, $s$)'s quantity descriptor is `accumulator`, then, for each input UTXO containing  ($g$, $s$) with quantity $n$ , it exists one output containing  ($g$, $s$) with quantity  $m \geq n$ .
+  - If  ($g$, $s$)'s quantity descriptor is `fractionable`, then, for each input UTXO containing  ($g$, $s$) with quantity $n$ ,  one or more outputs containing  ($g$, $s$) whose sum is equal to $n$.
+
+#### Merging Assets
+
+Merging assets mean that we take to assets that are fungible at one level but not the other and merge them into the same UTXO.  This requires the submission of a merging transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction.

```

Comment:
`take to assets` should be `take two assets` or possibly `take two (or more) assets`